packages/core/src/fhir-deduplication/__tests__/group-same-diag-reports.test.ts

```diff
@@ -49,11 +123,62 @@ describe("groupSameDiagnosticReports", () => {
     expect(diagReportsMap.size).toBe(2);
   });
 
+  it("discards diagReport if result and presented form are not present", () => {
+    diagReport.effectiveDateTime = dateTime.start;
+    diagReport.performer = [practRef];
+
+    const { diagReportsMap } = groupSameDiagnosticReports([diagReport]);
+    expect(diagReportsMap.size).toBe(0);
+  });
+
+  it("keeps diagReport if result is present and presentedForm is not - without a date", () => {
+    diagReport.performer = [practRef];
+
+    diagReport.result = resultExample;
```

Comment:
Where's the delete `diagReport.presentedForm`? And it would be good to explicitly set the date as undefined or delete it.

Reply:
put it on the `beforeEach`